Geometry_Engine/Query/IsContaining.cs

```diff
@@ -709,6 +709,182 @@ public static bool IsContaining(this Cuboid cuboid, IGeometry geometry, bool acc
             return cuboidBox.IsContaining(geoBox, acceptOnEdge, tolerance);
         }
 
+        /***************************************************/
+        /**** Public Methods - Closed Volume Polylines  ****/
+        /***************************************************/
+
+        [Description("Checks if a point lies within a collection of polylines describing a closed volume using their primitive planes and bounds.")]
+        [Input("panels", "A collection of Polylines outlining each of the faces of the closed volume to check with.")]
+        [Input("point", "The point to check to see if it is contained within the bounds of the panels.")]
+        [Input("acceptOnEdges", "Decide whether to allow the point to sit on the edge of the panel, default false.")]
+        [Input("tolerance", "Distance tolerance to use to determine intersections.")]
+        [Output("isContaining", "True if the point is contained within the bounds of the panels, false if it is not for each point provided.")]
+        public static bool IsContaining(this List<Polyline> panels, Point point, bool acceptOnEdges = false, double tolerance = BH.oM.Geometry.Tolerance.Distance)
```

Comment:
Calling this input `panels` might be confusing to standard geometry users. I'm wondering if we call it something like `closedVolume` instead to indicate it's a collection of polylines representing a closed volume (so helping with the description)?

Reply:
Agreed with @FraserGreenroyd - I would actually consider going a step further and suggest using `BoundaryRepresentation` as input, indicating we are speaking of a closed volume here. However, this would only make sense if converting the outlines to a Brep does not introduce any friction into existing code ofc.

Reply:
Renamed, leaving as closedVolume for now, a followup issue/pr can be created if we want to move from current handling to a proper brep based method for closed volumes.